cli/src/commands/touch.rs

```diff
@@ -74,6 +74,14 @@ pub(crate) struct TouchArgs {
         value_parser = parse_author
     )]
     author: Option<(String, String)>,
+
+    /// Preserve the committer
+    #[arg(long)]
+    preserve_committer: bool,
+
+    /// Preserve the committer timestamp
+    #[arg(long)]
+    preserve_committer_timestamp: bool,
```

Comment:
Do we need these options? Can you include supposed use cases in the commit description?

If we add these options, I think we'll need edge case handling for noop changes.

Reply:
An example use case could be fixing an author name in a repository without making the committer appear everywhere. But that's a quite limited usage.
A more useful use case could be removing a copyrighted file from a repository without altering the committer and committer timestamp of a large number of commits in the repository.

So I'm thiking this kind of flag could be better at the root level command.

Reply:
@yuja do you have an opinion on having these flags in the root jj command? If it's ok for you, I'll remove that commit from this PR and prepare another PR for that.

Reply:
I wouldn't add these to global nor `jj touch` options because the committer timestamp isn't usually controlled by the user. We also rely on the committer timestamp to create new commit (with different change id or predecessor ids.)

Reply:
> I wouldn't add these to global nor jj touch options because the committer timestamp isn't usually controlled by the user.

This ought to give better control over things for which users have very little.

Git has the `GIT_COMMITTER_DATE` environment variable, or the `git rebase --committer-date-is-author-date` option.

If that's a problem for jj, we can always rely on git for the rare cases where this is needed.

> We also rely on the committer timestamp to create new commit (with different change id or predecessor ids.)

Do I risk breaking something in my repository if I edit the committer timestamp using git?

Reply:
I've removed anything committer-related.